### ISSUE TITLE
[7-1-3] Natural Wonder Placement does not search through water of the Seas/NW priority problem

### ISSUE THREAD

**Issue by adaneslavo** [LoneGazebo/Community-Patch-DLL#8164]
See this post:
https://forums.civfanatics.com/threads/7-1-new-beta-version.671286/page-9#post-16130219
I described here with many examples 2 problems:
1. AssignStartingPlots.lua file doesn't search through open sea waters when looking for some place for Natural Wonders;
2. The priority marker in NaturalWonderPlacement table is not working as intended because even if high priority NW found the matching spots, it is not placed, because it is far away in order.

**Comment by LoneGazebo** [comment]
@azum4roll is I think the map map here. Or tu_79

**Comment by adaneslavo** [comment]
We found out, that the problem lied in latest Fertility changes in AssignStartingPlots.lua file. It neglected Sea tiles, treating them as "weak" or "no fertile". I modified this in my own file that overrides based VP one, but...

I'm curious how Great Barrier Reef spawn rate performs in base VP, if all Sea (Water) tiles are weak, and it is water-based Natural Wonder? I haven't play base VP for a long time, so I cannot answer this question, but that's just my loose thought.
